Cython/Compiler/PyrexTypes.py

```diff
@@ -2869,6 +2870,7 @@ def __init__(self, return_type, args, has_varargs = 0,
         self.is_static_method = is_static_method
         self.templates = templates
         self.is_strict_signature = is_strict_signature
+        self.op_arg_struct = None
```

Comment:
The pattern used everywhere else is to assign the default value to a class attributes (and then assign any new value as an instance attribute). I think the advantage is because it saves a bit of memory if you don't override the default.

So this gets moved to line ~2851

Reply:
Ah, that's clever. Fixed.